layouts/almalinux-day-germany-2024/single.html

```diff
@@ -346,7 +359,7 @@ <h4 style="font-size: 23px; letter-spacing: 0.80px; font-weight: 500; margin-bot
             </span>
             <div class="schedule-photo" style="display: flex;">
```

Comment:
```suggestion
            <div class="schedule-photo" style="display: flex; flex-wrap: wrap;">
```
As suggested by sboldreva